tests/visual-regression.spec.ts

```diff
@@ -0,0 +1,43 @@
+import path from 'node:path';
+import { test, expect } from '@grafana/plugin-e2e';
+import dashboardJson from '../provisioning/dashboards/dashboard.json';
+
+const panelScreenshotOptions = {
+  maxDiffPixels: 0,
+  stylePath: path.join(__dirname, 'show-panel-only.css'),
```

Comment:
🚫 👀 This CSS helps us to hide the non-panel dashboard elements -- since we don't want our tests to break when minor changes are made to dashboard elements.